features/step_definitions/where_did_you_hear_about_us_report_steps.rb

```diff
@@ -0,0 +1,28 @@
+Given(/^there are existing where did you hear about us records$/) do
+  create_list(:where_did_you_hear, Kaminari.config.default_per_page + 1)
```

Comment:
My view on this is that everyone knows exactly what `create`, `build`, `create_list` mean in the context of a test so the prefix is noise. Why are these any different to other globally scoped things like `subject`, `let`, `before` etc? The global namespace in the context of tests is polluted to heck anyway.